doc/source/api/python/python_examples.myst

```diff
@@ -0,0 +1,99 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Examples
+
+## Getting information on a raster dataset using dedicated methods
+
+The following snippet uses individual API methods to retrieve characters of
+a GDAL raster dataset and its bands.
+
+```{code-cell}
+
+from osgeo import gdal
+import json
+
+gdal.UseExceptions()
+
+with gdal.Open("data/byte.tif") as ds:
```

Comment:
I would try to limit the cells to 6-10 lines (excluding imports etc.) so that outputs are more easily related to the code that produced them.

Reply:
I couldn't find a way to split a cell in several parts, with continuation, especially with the use of the ``with`` syntax

Reply:
The `with` block can't be split between cells, but otherwise each cell starts picks up with the state of the previous cell.

![image](https://github.com/user-attachments/assets/f85e01bc-248f-4cff-9a1b-86211ac8960a)

Reply:
split as suggested, and added a paragraph in dev_documentation.rst about it